articles/build-an-outlook-clone-using-react-hooks/index.md

```diff
@@ -0,0 +1,895 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: build-an-outlook-clone-using-react-hooks
+title: Building Outlook Clone with React Hooks
+description: This article helps developers get started with React by building an Outlook clone using React so that you can learn the fundamentals by getting hands-on with the code.
+author: lalithnarayan-c
+date: 2020-08-09T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/build-an-outlook-clone-using-react-hooks/hero.jpg
+    alt: output image outlook clone example
+---
+Welcome to part two of the React series. In this tutorial, we will cover the same outlook clone that we built in [part one](/engineering-education/build-an-outlook-clone-using-react), but this time we will use React Hooks. The earlier codebase was repetitive and there is a scope for optimization of the entire project. To skip the theory and get straight into coding, [click here](#Code)
```

Comment:
the click here tag does not redirect - not sure if I messed anything up - please check and revise.  👍 @lalith1403

Reply:
It is working! Both on my PC as well as on GitHub. 

Check this link: https://github.com/section-io/engineering-education/blob/abeb74e146e22158033f152d1107559616c6dadb/articles/build-an-outlook-clone-using-react-hooks/index.md

The link works...